tools/noxfile.py

```diff
@@ -0,0 +1,18 @@
+import nox
```

Comment:
elderly me has difficulty remembering which project uses what for testing.
I wonder if there was a *need* to use `nox` here (not that I don't like having python config etc, but otherwise not familiar with it) instead of `tox`?

Reply:
tox should really only be used for projects that can be installed (i.e., that have a `setup.py` etc.).  nox has no such requirement.